package.json

```diff
@@ -4,20 +4,41 @@
   "description": "A library of components for accessing RESTful services with javascript/typescript.",
   "author": "David de Regt <David.de.Regt@microsoft.com>",
   "scripts": {
-    "prepublish": "tsc",
+    "prepare": "tsc",
+    "clean": "rimraf dist",
     "build": "npm run tslint && tsc",
-    "tslint": "tslint --project tsconfig.json -r tslint.json --fix || true"
+    "tslint": "tslint --project tsconfig.json -r tslint.json --fix || true",
+    "test": "npm run clean && karma start --singleRun",
+    "test:watch": "npm run clean && karma start"
   },
   "dependencies": {
-    "@types/lodash": "^4.14.62",
-    "assert": "^1.4.1",
-    "lodash": "4.x",
-    "synctasks": "^0.3.0"
+    "@types/lodash": "4.14.107",
+    "assert": "1.4.1",
+    "lodash": "4.17.5",
```

Comment:
Need to leave this as 4.x, or at least ^4.17.0 -- you're prescribing a specific lodash here.

Reply:
thanks, will change